deb/openmediavault/usr/share/openmediavault/engined/rpc/usermgmt.inc

```diff
@@ -142,6 +142,26 @@ finish:
 		];
 	}
 
+	/**
+	 *  Detect if system is using SSSD for user authentication
+	 */
+	private function isUsingSSSD() {
+		// Check if the sssd service is active
+		$cmd = new \OMV\System\Process("systemctl", ["is-active", "sssd"]);
```

Comment:
Please make use of the [systemctl](https://github.com/openmediavault/openmediavault/blob/41c940eda200c4c1810bb06d51a696f2b31040ae/deb/openmediavault/usr/share/php/openmediavault/system/systemctl.inc#L30) wrapper. The helper class contains an [isActive](https://github.com/openmediavault/openmediavault/blob/41c940eda200c4c1810bb06d51a696f2b31040ae/deb/openmediavault/usr/share/php/openmediavault/system/systemctl.inc#L191) method.